### DISCUSSION TITLE
FreshRSS on Mastodon

### DISCUSSION THREAD

**Discussion by math-GH** [FreshRSS/FreshRSS#4809] [Ideas]
Hi,
do we want to have a FreshRSS account on Mastodon?

The hashtag is already used :) https://mastodon.social/tags/freshrss

**Comment by Alkarex** [comment]
I had a (self-hosted) fediverse account for me, but I have not reinstalled it after my old server hard-disk crashed. Maybe I will restart an instance.
Marien also has an account somewhere.

**Reply by math-GH** [reply]
floss.social could be also a good server that would fit to us

**Comment by math-GH** [comment]
see here too: https://github.com/FreshRSS/FreshRSS/issues/6079

**Comment by Alkarex** [comment]
Done here https://framapiaf.org/@freshrss

@math-GH Do you have a personal account to follow and potentially re-share FreshRSS-related articles?

P.S.: By that I mean that the FreshRSS account could re-share some of your FreshRSS-related posts.

**Reply by math-GH** [reply]
I am just reading there. No content.

**Reply by Alkarex** [reply]
What about @Frenzie , @aledeg , @Inverle : do you have any Mastodon or Fediverse account?

**Reply by Frenzie** [reply]
I have one at https://social.vivaldi.net/@Frenzie

**Reply by aledeg** [reply]
I don't

**Reply by Inverle** [reply]
No
